analyzer_test.go

```diff
@@ -75,7 +76,7 @@ func TestAnalyzer_fix(t *testing.T) {
 		{dir: "fix", cfgFilename: "fix.yml"},
 		{dir: "sample", cfgFilename: "sample.yml"},
 		{dir: "noheader", cfgFilename: "noheader.yml"},
-		{dir: "regexpvalue_issue", cfgFilename: "regexpvalue_issue.yml"}, // FIXME BUG
+		// {dir: "regexpvalue_issue", cfgFilename: "regexpvalue_issue.yml"}, // FIXME BUG
```

Comment:
@ldez To fix this case, I think we need to define default values for variables that can be used on generating a fix. 
Thoughts?

Reply:
The problem is not the variables, but the regexp matching.

Reply:
OK, going debugging. Thanks for information.

Reply:
The test is just an example.
You can create a case with only one regexp without any other variables, and you will have the same problem because the problem is that when the regexp does not match, it's not possible to suggest a fix.

Reply:
Yeah, and that's why i'm thinking of adding default values. 

For this case config could look something like this:
```yaml
template: |-
  {{.COPYRIGHT_HOLDER}}TEXT

values:
  regexp:
    YEAR: '(2020)|(2021)'
    COPYRIGHT_HOLDER: |-
      (A {{ .YEAR }}
      (.*)
      )+
 default:
   COPYRIGHT_HOLDER: |-
   A {{ .YEAR }}
   B
   A  {{ .YEAR }}
   C
```

Also, probably this problem is a bit out of scope for the release, but if you're fine with default values, we can go ahead and implement it.

Reply:
The problem is the suggested fixes: the current code will destroy headers.
This is a major problem.

From my memories, this problem didn't happen before, so this is an important regression.

The "default value" is weird.

Reply:
> The problem is the suggested fixes: the current code will destroy headers.

LOL. You're right. As I recall it's not a regression, we've tried to use a golangci-linter -fix with regexp variable on a rich project. And it did not fix variables well (just pasted regexp), so it's an existing problem of regexp values.

Reply:
So if the problem exists in the 0.5.0 version, my suggestion then is consider it in next version (v1.1.0)

Reply:
I've quickly checked and with golangci-lint v1.63 fix option just adds regexp. So I think the problem should be considered separately.